### ISSUE TITLE
Switch Ionic app to use a different Mobile Hub project?

### ISSUE THREAD

**Issue by ryaa** [aws/awsmobile-cli#131]
I created a new Ionic mobile application using this starter template - see https://github.com/ionic-team/starters/tree/master/ionic-angular/official/aws. During setup it created a new Mobile Hub project and associated the ionic app with it (the appropriate details were recorded into awsmobilejs directory files and src/aws-export.js).
The problem is that there is already a Mobile Hub project configured with the existing Cognito Pool which needs to be used by this ionic app. I manually changed src/aws-export.js file params to use it which worked fine until i needed to publish the app to S3. After running awsmobile publish (which also pulled the backend config) all the manual settings were replaced.

Question - is there anyway to switch the existing mobile app to a different Mobile Hub project? I reviewed awsmobile cli commands and issue tracker and could not find how to do this.

**Comment by UnleashedMind** [comment]
Yes, by executing the init command with the project id of the Mobile Hub project that you wanted to switch to. 
`awsmobile init <mobile-hub-project-id>`
you can find the mobile hub project id after you click open the mobile hub project in the web console, and the id will be in the url. 
`https://console.aws.amazon.com/mobilehub/home?#/<mobile-hub-projectid>/build`

**Comment by ryaa** [comment]
The task has been complete. aws-amplify project has been re-initialized and synchronized with the backend. The latest version of the app is published and hosted at:
http://couriersprod-hosting-mobilehub-695662618.s3-website.eu-central-1.amazonaws.com
and the application is also distributed through aws CloudFront
https://d31focr8ouhvry.cloudfront.net

However login and other features don't work because of CORS issue. I added a separate ticket to resolve this https://github.com/frisches/logistics-backend/issues/105
